src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSCloudTask.cs

```diff
@@ -401,7 +401,8 @@ public IList<PSResourceFile> ResourceFiles
                 this.resourceFiles = value;
             }
         }
-        
+
+        [Obsolete("This will be removed in a future version and replaced with UserIdentity")]
```

Comment:
@matthchr for these warnings, can we say "Property <name> will be removed..." instead of "This will be removed..."

This will help users with scripts easily identify where the warning message is coming from in their script.